README.md

```diff
@@ -246,9 +246,13 @@ Add a link reference for the GitHub "compare" view at the bottom of the CHANGELO
 
 ### 6. Create a pull request against the master branch
```

Comment:
```suggestion
### 6. Create a pull request against the develop branch
```

Reply:
I guess if you're fixing this, maybe add the step to merge master back into develop?

Reply:
@flamingbear I switched the steps around so this won't happen again. First you merge to develop now, then to master, so develop will have everything already. I think that makes more sense as a promotion path anyway.

@ifestus Committed your suggestion, thank you!